org.freedesktop.Sdk.Extension.php73.json

```diff
@@ -0,0 +1,146 @@
+	"branch": "18.08",
+	"runtime": "org.freedesktop.Sdk",
+	"build-extension": true,
+	"sdk": "org.freedesktop.Sdk",
+	"runtime-version": "18.08",
+	"sdk-extensions": [],
+	"separate-locales": false,
+	"appstream-compose": false,
+	"cleanup": [
+		"/share/info",
+		"/share/man",
+		"*.a",
+		"*.la"
+	],
+	"build-options": {
+		"no-debuginfo": true,
+		"strip": true,
+		"prefix": "/usr/lib/sdk/php73"
+	},
+	"modules": [
+		{
+			"name": "libzip",
+			"buildsystem": "cmake-ninja",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://libzip.org/download/libzip-1.5.2.tar.gz",
+					"sha256": "be694a4abb2ffe5ec02074146757c8b56084dbcebf329123c84b205417435e15"
+				}
+			]
+		},
+		{
+			"name": "php",
+			"buildsystem": "autotools",
+			"config-opts": [
+				"--enable-re2c-cgoto",
+				"--with-tsrm-pthreads",
+				"--enable-phpdbg",
+				"--disable-cgi",
+				"--with-argon2",
+				"--enable-exif",
+				"--enable-ftp",
+				"--enable-mbstring",
+				"--enable-intl",
+				"--enable-calendar",
+				"--enable-dba=shared",
+				"--enable-bcmath",
+				"--enable-bz2",
+				"--enable-ctype",
+				"--enable-curl",
+				"--enable-date",
+				"--enable-dba",
+				"--enable-dom",
+				"--enable-enchant",
+				"--enable-fileinfo",
+				"--enable-filter",
+				"--enable-gd",
+				"--enable-gettext",
+				"--enable-gmp",
+				"--enable-hash",
+				"--enable-iconv",
+				"--enable-imap",
+				"--enable-intl",
+				"--enable-json",
+				"--enable-ldap",
+				"--enable-libxml",
+				"--enable-mysqlnd",
+				"--enable-opcache",
+				"--enable-openssl",
+				"--enable-pcntl",
+				"--enable-pcre",
+				"--enable-pdo",
+				"--enable-pdo_mysql",
+				"--enable-pdo_pgsql",
+				"--enable-pdo_sqlite",
+				"--enable-phar",
+				"--enable-posix",
+				"--enable-pspell",
+				"--enable-readline",
+				"--enable-recode",
+				"--enable-reflection",
+				"--enable-session",
+				"--enable-simplexml",
+				"--enable-snmp",
+				"--enable-sockets",
+				"--enable-sodium",
+				"--enable-sqlite3",
+				"--enable-sysvmsg",
+				"--enable-sysvmem",
+				"--enable-sysvshm",
+				"--enable-tidy",
+				"--enable-tokenizer",
+				"--enable-wddx",
+				"--enable-xml",
+				"--enable-xmlreader",
+				"--enable-xmlrpc",
+				"--enable-xmlwriter",
+				"--enable-xsl",
+				"--with-libzip",
+				"--enable-zlib",
+				"--with-zlib",
+				"--with-openssl"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.php.net/distributions/php-7.3.5.tar.gz",
+					"sha256": "c953749b7f3310a3a74f920ef698f6d1c04636d11656ac9ffb3ab10d34e30e1e"
+				}
+			]
+		},
+		{
+			"name": "php.ini",
+			"buildsystem": "simple",
+			"build-commands": [
+				"cp php.ini /usr/lib/sdk/php73/lib/php.ini"
+			],
+			"sources": [
+				{
+					"type": "file",
+					"path": "php.ini-development",
+					"dest-filename": "php.ini"
+				}
+			]
+		},
+		{
+			"name": "composer",
+			"build-options": {
+				"build-args": ["--share=network"]
```

Comment:
Builder don't have access to network after downloading the sources. This won't work here on Flathub.

Reply:
I'll try to work it out without that then.